src/components/Editor.js

```diff
@@ -127,9 +128,23 @@ const Editor = React.createClass({
       click: () => this.props.jumpToMappedLocation(sourceLocation)
     };
 
-    showMenu(event, [
+    const menuOptions = [
       jumpLabel
-    ]);
+    ];
+
+    if (this.editor.codeMirror.somethingSelected()) {
+      const watchExpressionLabel = {
+        accessKey: "E",
+        disabled: false,
+        label: L10N.getStr("editor.watchExpression"),
+        click: () => this.props.addExpression({
```

Comment:
> lets move this to an instance function as we probably want to also check to see if the cursor is in the selection

couldn't find a straightforward way of doing it. basically, the cursor stays within the selection even though the pointer is right-clicking somewhere else. will follow-up on this in a subsequent PR.